src/kibana/components/index_patterns/index_patterns.js

```diff
@@ -11,6 +11,7 @@ define(function (require) {
     var patternCache = Private(require('components/index_patterns/_pattern_cache'));
 
     var notify = new Notifier({ location: 'IndexPatterns Service'});
+    var DEFAULT_INDEX_PATTERN = '[network_]YYYY_MM_DD';
```

Comment:
All uppers is pretty common for constants. We're not using ES6 javascript yet so there's no built-in way to distinguish between constants and variables.